### PR TITLE
Release/uan 2.3

### PR DESCRIPTION
**Pull request by darrenn-jackson**
## Summary and Scope

Added several files: 
   - XML manifests for both the Admin and Install Guides
   - JSON publication metadata files for both Guides
   - A build script that produces HTML5 output for HPESC, copies in the correct JSON, and zips up a bundle for the guides
   - Added a few empty files for section grouping in the XML manifests to mirror the structure of the current .mds files

This change enables us to publish our Markdown docs content directly to HPESC, allowing us to do away with PDF deliverable (PDFs can be generated from HPESC). More importantly, this replaces the current broken PDF build process and allows us to produce docs for the upcoming HPE Cray EX release.

This is a backwards incompatible fix for generating docs, but the previous docs build process was already broken. This only affects building docs.

## Issues and Related PRs

* Resolves [TPCCMS-230](https://jira-pro.its.hpecorp.net:8443/browse/TPCCMS-230)
* Change will also be needed in `main`

## Testing

This was tested in my WSL2 Ubuntu environment with DITA-OT 3.7 and the HPE ITG test portal. Here are the links to both guides currently published there:
   - https://itg-int.support.hpe.com/hpesc/docDisplay?docId=crs8032_1en_us
   - https://itg-int.support.hpe.com/hpesc/docDisplay?docId=crs8033_1en_us

### Test description:

Changes tested by completing the KM Direct Publishing process with the HTML5 zip bundles and inspecting the published content.

## Risks and Mitigations

Since the header levels were changed in all the Markdown source files, the scripts for Hugo processing will probably need to be modified. DITA-OT requires that all source Markdown files:
   - Start with a level one header
   - Do not transition to a header level lower the immediate next one down. In other words, changing immediately from a level one header to a level three header is not allowed.

## Pull Request Checklist

- Version number NOT incremented
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- No [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH)

### CODE REVIEW

File: docs/portal/developer-portal/upgrade/Notable_Changes.md [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
-#### UAN 2.3.1
+## UAN 2.3.1
 
 * UAN 2.3 no longer ships a default recipe or image. To build a UAN image, administrators should select a COS recipe to build as the base of their UAN and Application Nodes.
 * The role `uan_packages` will now install the rpms needed by UAN and Application Nodes
 * The role `uan_packages` role supports GPG checking when the CSM version is 1.2 or greater.
   * `uan_disable_gpg_check: yes` must be set if CSM is earlier than 1.2
   * `uan_disable_gpg_check: no` should be set if CSM is 1.2 or greater
 
+## UAN 2.3.2
```
Comment (alanm-hpe) [inline docs/portal/developer-portal/upgrade/Notable_Changes.md:19]: There was nothing notable in 2.3.2 from 2.3.1 other than newer COS fixes. There is also none of the features from below related to BiCAN and CHN. I think the whole section should be removed.

**Review by alanm-hpe — COMMENTED** [review]
I'll approve once the section about BiCAN/CHN is removed from Notable Changes. That was not delivered in 2.3.2

**Review by alanm-hpe — APPROVED** [review]

**Comment by alanm-hpe** [comment]
@keopp-hpe ok to merge? Is there anything you wanted to look at from a `hugo` perspective?

**Review by keopp-hpe — APPROVED** [review]

**Comment by keopp-hpe** [comment]
There will be some changes needed to the hugo docs generation code, but I'll sort that in another PR after this one merges.

**Pull request merged**